Dockerfile.ui-dashboards

```
@@ -31,7 +31,7 @@ ENV CGO_ENABLED=1
 
 RUN make build-backend BUILD_OPTS="-tags strictfipsruntime"
 
-FROM registry.redhat.io/rhel9-2-els/rhel:9.2
+FROM registry.redhat.io/rhel9-2-els/rhel:9.4
```

@marioferh  I think this should be:
```suggestion
FROM registry.redhat.io/rhel9-4-els/rhel:9.4
```

---

right